src/dom/Draggable.js

```diff
@@ -75,7 +75,7 @@ export var Draggable = Evented.extend({
 
 		// If we're currently dragging this draggable,
 		// disabling it counts as first ending the drag.
-		if (L.Draggable._dragging === this) {
+		if (this._dragging === this) {

```

Comment:
This also highlights the fact that the comment is misleading, since it's not "this draggable" we're checking, but actually any draggable.